frontend/src/components/TransactionEventList/Events/PltTransferMemo.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+	<div
+		v-if="memo?.__typename === 'Memo'"
+		class="bg-gray rounded flex items-center gap-2"
+	>
+		<span>Memo:</span>
+		<span class="break-all whitespace-pre-line text-xs">{{
+			memo?.bytes ? Buffer.from(memo.bytes, 'hex').toString('utf-8') : ''
```

Comment:
A `memo` can be either cbor encoded data or text data. You are only handling the text data here. The backend already decoded the `memo` for `RegisterData` tx and `CCDTransfer` tx. Please use that decoding:

https://github.com/Concordium/concordium-scan/blob/fa0b4f2681418b05e22cf028772036462d55230e/backend/src/decoded_text.rs#L4